### PR TITLE
created the workspace dashboard

### PR DESCRIPTION
**Pull request by samanthabaker**
I've added the layout and the components needed. 
I still need to move the style sheet to sam styles and finish the Role Request components.

### CODE REVIEW

File: src/patterns/pages/workspace-dashboard/workspace-dashboard.template.html [hunk 1]
```diff
@@ -1,15 +1,140 @@
...
+  <div class="sidenotif">
+    <!-- Profile -->
+    <sam-card>
+      <ng-container class="header">
+        <h3 class="sam heading">
+          <a class="sam semibold">Profile</a>
+        </h3>
+      </ng-container>
+      <div class="sam feed visible tablet up">
+        <div class="profileicon">
```
Comment (thomastighe) [inline src/patterns/pages/workspace-dashboard/workspace-dashboard.template.html]: Do we allow direct path to source for images?

File: src/patterns/pages/workspace-dashboard/workspace-dashboard.template.html [hunk 2]
```diff
@@ -141,7 +373,45 @@ <h3 class="sam heading middle aligned">
...
       </div>
     </sam-segment>
-    
+
+    <!-- FEDERAL HIERARCHY -->
+    <sam-segment>
+      <h3 class="sam heading middle aligned">
+        <!-- SVG: Created by Henry Chung from the Noun Project -->
+        <span class="icon">
+          <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" x="0px" y="0px" viewBox="0 0 100 125" style="enable-background:new 0 0 100 100;" xml:space="preserve">
```
Comment (thomastighe) [inline src/patterns/pages/workspace-dashboard/workspace-dashboard.template.html]: Is this how we handle svg?

File: src/patterns/pages/workspace-dashboard/workspace-dashboard.template.html [hunk 3]
```diff
@@ -1,147 +1,455 @@
...
+    <div class="sam search cards">
+
+      <!-- USER DIRECTORY -->
+      <div class=" user search">
+        <sam-segment>
+          <h3 class="sam heading middle aligned">
+            <!-- SVG: Created by Henry Chung from the Noun Project -->
+            <span class="icon">
+              <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" x="0px" y="0px" viewBox="0 0 100 125" style="enable-background:new 0 0 100 100;" xml:space="preserve">
+                <path d="M77.5,60C67.8,60,60,67.8,60,77.5C60,87.2,67.8,95,77.5,95S95,87.2,95,77.5C95,67.8,87.2,60,77.5,60z M77.5,91.9  c-7.9,0-14.4-6.5-14.4-14.4c0-7.9,6.5-14.4,14.4-14.4s14.4,6.5,14.4,14.4C91.9,85.4,85.4,91.9,77.5,91.9z"/>
```
Comment (carlosvalle) [inline src/patterns/pages/workspace-dashboard/workspace-dashboard.template.html]: Now that we have all svg inside the assets folder - could you replace this with the svg path

File: src/patterns/pages/workspace-dashboard/workspace-style.scss [hunk 4]
```diff
@@ -0,0 +1,312 @@
+.workspace.dash{
```
Comment (carlosvalle) [inline src/patterns/pages/workspace-dashboard/workspace-style.scss]: Remove this file - this code is already in sam-styles/temp

**Pull request merged**